torch_frame/data/stats.py

```diff
@@ -145,9 +163,22 @@ def compute_col_stats(
             for stat_type in StatType.stats_for_stype(stype)
         }
     else:
+        if stype == torch_frame.timestamp:
+            ser = pd.to_datetime(ser, format=time_format)
+            ser = ser.sort_values()
         stats = {
-            stat_type: stat_type.compute(ser.dropna(), sep, time_format)
+            stat_type: stat_type.compute(ser.dropna(), sep)
             for stat_type in StatType.stats_for_stype(stype)
         }
 
     return stats
+
+
+def extract_time(t: datetime) -> Tensor:
+    # subtracting one so that the smallest months and days can
+    # start from 0.
+    times = [
+        t.year, t.month - 1, t.day - 1,
+        t.weekday(), t.hour, t.minute, t.second
+    ]
+    return torch.tensor(times)
```

Comment:
Is there any way to call `TimestampTensorMapper` here, or use some shared function, similar to how we reuse https://github.com/pyg-team/pytorch-frame/blob/master/torch_frame/data/mapper.py#L140-L154 for multi-categorical?

Reply:
This way, we keep the ordering in the same unified place.